packages/amplication-data-service-generator/project.json

```diff
@@ -36,7 +36,7 @@
       ],
       "outputs": ["{projectRoot}/node_modules"],
       "options": {
-        "commands": ["npm ci"],
+        "commands": ["npm ci", "ts-node ./scripts/update-version.ts"],
```

Comment:
I suggest adding this as a separate target to the project.json and call it as dependence. What do think?

Reply:
I don't think it is necessary. What are the benefits? Do we need this command elsewhere?

Reply:
I think it's important to separate commands for there proper use; in that case, I will add `update-version` script that runs prebuild, preserve and prepublish.